ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/Rectangle selection/rectangle-selection-tool.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { Page, test } from '@playwright/test';
+import {
+  BondTool,
+  BondType,
+  TopPanelButton,
+  clickOnAtom,
+  dragMouseTo,
+  drawBenzeneRing,
+  getControlModifier,
+  getCoordinatesOfTheMiddleOfTheScreen,
+  getCoordinatesTopAtomOfBenzeneRing,
+  openFileAndAddToCanvas,
+  selectNestedTool,
+  selectTopPanelButton,
+  takeEditorScreenshot,
+  waitForPageInit,
+} from '@utils';
+import { getAtomByIndex } from '@utils/canvas/atoms';
+import { getBondByIndex } from '@utils/canvas/bonds';
+
+test.describe('Rectangle selection tool', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+  });
+
+  test.afterEach(async ({ page }) => {
+    await takeEditorScreenshot(page);
+  });
+
+  const xDelta = 30;
+  const yDelta = 60;
+  const modifier = getControlModifier();
+
+  const xMark = 300;
+  const yMark = 200;
+
+  async function selectObjects(page: Page, xAxis: number, yAxis: number) {
```

Comment:
May be we can use something like `xAxisRadius` or something?

Reply:
done